_locales/en/messages.json

```diff
@@ -5,6 +5,9 @@
   "reloadLiveBookmark": {
     "message": "Reload Live Bookmark"
   },
+  "openAllUnreadPages": {
+    "message": "Open all unread pages in tabs"
```

Comment:
I think maybe we should s/pages/feeds/ or s/page/live bookmarks/.

Reply:
Sorry, i do not understand what do you mean